dr_ensenso_node/src/ensenso.cpp

```diff
@@ -483,6 +527,36 @@ class EnsensoNode: public ros::NodeHandle {
 		return true;
 	}
 
+	bool onGetCameraParams(dr_ensenso_msgs::GetCameraParams::Request & req, dr_ensenso_msgs::GetCameraParams::Response & res) {
+		if (true
+			&& (req.camera) != dr_ensenso_msgs::GetCameraParams::Request::MONO
+			&& (req.camera) != dr_ensenso_msgs::GetCameraParams::Request::LEFT
+			&& (req.camera) != dr_ensenso_msgs::GetCameraParams::Request::RIGHT
```

Comment:
Could you remove the parenthesis around `(req.camera)`?

If you do that and change the unique_ptr thing above, we're ready to merge :)